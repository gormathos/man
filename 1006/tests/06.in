C1;C2;C3;C4;C5;C6;C7;C8;C9;C10;C11;C12;C13;C14;C15;C16;C17;C18;C19;C20;C21;C22;C23;C24;C25;C26;C27;C28;C29;C30;C31;C32;C33;C34;C35;C36;C37;C38;C39;C40;C41;C42;C43;C44;C45
15;73;20;61;41;16;23;47;20;27;46;76;53;57;23;28;32;63;43;49;13;21;66;24;52;57;45;32;36;69;8;45;46;41;9;31;62;11;16;43;56;65;62;27;25
74;11;19;35;70;55;31;14;9;41;39;60;64;31;21;57;13;18;48;20;23;60;18;18;54;23;19;12;76;31;39;73;39;32;74;74;61;41;19;21;55;23;52;72;73
58;13;25;11;12;13;41;37;37;11;62;41;24;13;18;19;48;71;75;47;43;56;45;28;50;43;48;67;43;50;18;34;21;37;25;22;19;50;45;56;30;41;34;64;25
12;15;25;31;52;73;68;70;75;44;36;36;69;21;54;56;69;34;72;28;58;57;52;75;69;25;65;9;76;24;39;45;75;45;67;12;40;39;63;58;58;34;43;44;71
57;75;12;34;42;15;49;15;15;75;61;10;39;58;39;74;54;36;30;13;47;45;36;65;69;57;22;56;58;52;9;44;66;17;18;30;14;66;31;8;52;68;45;72;59
11;72;28;73;70;64;8;15;34;41;71;70;53;65;35;36;35;11;54;35;51;75;23;39;22;16;8;45;35;76;11;57;25;25;18;26;37;9;43;53;13;16;32;31;64
65;53;58;14;57;42;38;73;34;42;69;47;25;17;9;17;76;22;54;74;35;71;17;47;76;32;70;54;47;57;30;29;9;56;62;42;10;26;24;46;48;12;45;58;52
54;18;60;51;74;65;60;53;30;23;18;43;26;45;40;39;41;30;16;66;26;18;42;76;8;51;17;31;12;60;49;56;56;40;15;21;16;41;66;37;15;28;35;21;15
62;22;17;37;9;53;23;67;63;19;72;14;73;54;29;56;47;74;51;43;36;9;35;21;75;45;42;31;47;55;18;36;59;35;64;61;27;13;57;17;61;70;61;48;48
30;35;58;68;44;55;68;20;51;34;22;49;67;62;54;12;29;35;44;26;45;51;42;54;68;69;16;74;58;34;27;14;15;33;69;64;9;49;56;54;46;39;40;33;30
31;26;14;69;53;54;58;47;62;68;46;17;66;51;68;10;27;65;18;37;21;26;39;40;74;49;60;55;29;53;53;54;64;26;41;70;16;9;55;26;29;49;43;25;57
73;66;8;21;8;38;76;51;55;18;21;14;54;25;66;18;27;32;70;73;44;46;46;32;37;31;61;18;54;44;63;16;13;72;69;57;72;14;63;43;51;63;31;63;67
45;38;57;41;69;57;46;52;70;15;35;40;31;15;52;20;45;64;66;36;38;9;18;37;72;40;31;56;15;12;25;35;10;76;30;59;70;37;66;15;72;36;8;25;45
31;20;15;33;73;8;30;8;69;66;68;11;64;51;63;75;38;69;71;76;63;54;16;39;24;21;19;56;39;48;24;16;50;12;43;24;28;43;72;14;49;55;73;38;19
47;47;38;38;59;69;33;14;70;71;27;63;62;43;51;34;50;52;72;51;40;54;39;48;32;56;73;24;45;70;40;11;59;64;44;20;14;42;31;25;40;33;13;9;74
43;59;64;44;41;66;13;40;75;17;41;56;8;29;33;43;35;17;9;18;45;37;76;52;35;70;19;18;67;49;54;12;36;40;75;43;25;9;14;49;29;47;64;67;32
52;73;30;67;33;13;25;66;69;51;44;49;16;23;56;70;15;9;30;45;54;23;14;20;55;18;60;47;37;52;19;67;13;12;57;68;27;18;17;55;12;47;60;46;62
15;34;10;42;63;62;37;37;21;10;20;29;33;53;59;20;24;18;29;49;64;39;59;76;40;66;52;47;69;59;23;54;32;11;51;11;18;10;24;9;40;21;47;23;44
27;38;14;9;30;38;42;64;9;12;75;33;33;54;10;39;21;33;21;60;51;55;28;19;66;70;38;53;75;43;23;19;10;28;9;73;17;58;10;52;14;55;61;47;50
23;66;16;43;55;55;58;13;52;71;26;66;64;69;18;28;60;16;75;51;53;53;65;75;17;20;69;70;55;14;35;12;23;72;33;54;36;19;43;33;18;8;41;35;75
75;71;38;65;12;34;27;45;11;42;76;39;61;29;47;41;35;37;74;74;48;71;41;73;57;65;49;22;67;34;28;53;56;49;48;20;32;45;17;35;53;69;29;38;25
18;37;76;69;34;69;60;67;50;42;13;34;12;31;9;50;50;21;13;73;41;59;27;43;19;37;21;30;49;21;41;20;65;11;36;40;28;27;50;24;39;53;10;34;74
69;56;45;26;52;28;26;62;64;65;13;24;15;58;76;75;44;34;67;73;74;66;24;12;70;22;33;21;24;9;22;37;21;38;14;74;70;23;18;71;65;37;74;76;16
66;49;36;13;12;36;73;25;26;67;8;59;44;75;26;12;65;73;59;30;10;28;74;11;66;33;74;71;62;38;73;8;64;47;76;43;45;74;38;15;69;54;29;31;14
38;50;66;19;31;74;34;20;28;31;48;71;17;34;59;29;14;47;13;26;76;36;16;13;57;55;54;21;48;11;61;76;70;41;76;51;48;56;19;59;71;15;12;49;24
44;45;47;44;25;11;22;23;19;25;41;71;25;55;11;42;25;20;55;21;13;62;36;21;18;15;15;13;48;43;29;46;19;76;48;20;36;8;68;21;38;14;52;26;55
68;22;43;24;43;72;43;36;41;65;35;15;8;72;75;72;41;31;15;17;73;21;8;72;41;68;65;44;40;42;11;62;56;27;38;28;10;8;60;38;63;43;73;55;38
20;37;51;62;38;47;60;26;31;45;43;37;64;33;40;55;16;66;37;19;53;61;37;49;64;70;76;50;28;68;44;55;57;64;21;18;74;11;40;35;57;61;9;43;26
33;54;24;66;42;61;48;22;64;28;35;55;74;44;46;16;42;72;45;8;12;31;14;48;63;40;8;61;69;56;71;47;60;45;38;39;9;10;19;76;20;44;71;22;67
8;16;59;21;59;55;38;29;37;52;48;48;51;71;74;41;50;39;20;62;21;41;53;71;54;64;61;32;55;73;23;57;63;70;56;60;15;30;67;58;51;68;11;28;55
19;27;69;33;63;26;25;62;18;22;17;39;19;58;15;25;72;54;9;53;59;39;9;47;47;36;35;32;43;64;72;10;43;49;29;29;33;17;69;10;25;33;24;41;59
31;69;62;45;11;10;65;24;10;24;66;16;36;33;14;43;70;76;47;20;50;63;42;62;40;26;46;75;45;52;48;43;16;30;76;59;16;51;27;15;69;16;46;45;63
15;34;47;73;23;70;67;31;18;11;46;50;36;38;29;60;71;42;51;13;56;47;11;47;47;29;70;61;15;70;13;38;26;58;51;63;68;46;44;69;56;23;45;29;11
74;75;24;34;63;41;29;18;43;34;58;59;46;39;21;74;31;24;31;31;61;68;74;61;70;22;36;76;53;54;45;75;55;21;33;37;41;8;18;74;53;62;9;33;21
38;71;8;72;49;57;59;12;40;18;61;45;49;30;28;39;42;53;40;74;57;67;73;10;70;34;76;57;16;41;41;71;74;76;71;13;13;10;33;9;60;43;40;8;45
24;46;9;29;67;34;58;38;69;23;18;53;59;61;9;62;37;39;11;22;15;13;23;18;43;33;9;48;71;20;27;15;31;38;26;31;27;13;76;43;67;74;62;61;12
30;20;47;61;41;68;72;48;60;27;27;69;27;8;55;32;62;11;67;73;72;57;15;54;31;67;36;48;74;56;10;37;12;66;23;32;28;34;75;24;16;68;32;72;42
46;42;62;21;15;10;69;49;69;75;12;75;15;28;26;23;34;22;65;46;62;11;63;48;59;63;35;56;38;49;74;40;24;73;11;49;41;30;42;25;65;45;47;72;43
13;49;35;64;24;46;42;19;49;38;76;43;36;14;42;43;37;58;44;53;32;43;39;59;50;72;67;62;76;49;21;53;14;66;34;45;66;64;41;24;33;11;46;60;40
63;48;43;27;29;22;30;72;9;33;37;28;22;24;37;20;68;35;20;28;38;28;8;49;33;43;37;14;15;13;20;39;31;63;40;26;52;75;71;27;68;72;40;65;64
8;27;18;43;40;55;18;67;59;61;61;68;35;13;73;24;69;56;13;22;66;58;12;49;21;21;76;16;42;43;70;59;38;41;30;12;9;22;22;30;53;72;50;66;74
26;52;38;9;73;56;65;15;51;41;66;64;37;72;53;32;14;11;11;31;27;63;24;33;31;58;55;42;65;25;15;52;55;15;44;71;21;46;27;55;29;74;36;36;56
31;32;74;16;24;52;58;22;59;40;57;21;60;36;53;70;66;19;22;10;36;31;71;71;46;68;73;57;15;62;17;27;52;27;50;29;48;29;67;57;37;11;47;44;60
27;27;19;72;44;13;70;19;58;66;73;70;12;46;45;64;25;26;14;16;25;41;76;19;74;45;65;16;72;62;32;36;71;61;19;46;41;76;46;58;33;75;61;66;25
47;37;61;14;21;64;60;53;62;49;12;63;16;15;29;57;18;14;30;12;29;60;30;52;49;70;59;35;20;48;18;53;17;13;18;26;73;75;43;32;64;22;60;19;72
23;21;20;56;49;50;53;61;53;65;21;39;48;64;27;41;28;36;33;55;39;54;57;10;14;43;27;37;57;13;31;60;16;75;36;55;76;64;57;62;55;73;11;65;47
35;52;36;40;63;39;36;53;66;24;62;52;60;18;9;23;60;76;42;12;76;73;24;65;68;44;18;37;72;72;60;12;54;15;52;11;50;50;19;23;75;61;73;47;68
9;62;47;62;18;55;58;33;58;57;25;67;58;46;65;68;30;36;30;31;38;49;64;30;22;10;76;49;11;53;11;54;48;52;45;59;51;8;9;11;15;38;61;76;76
39;63;49;70;26;8;30;32;65;66;9;34;64;22;48;13;25;75;56;54;20;75;73;22;11;42;39;36;19;26;71;20;70;59;30;60;49;34;35;76;62;47;56;38;53
11;67;70;48;53;10;53;28;13;35;15;27;18;38;53;41;40;67;70;68;71;15;66;33;41;66;23;44;49;26;22;57;62;65;14;40;50;32;17;59;33;22;43;18;73
59;69;64;20;43;29;60;63;48;56;66;75;10;60;65;70;56;38;73;76;38;73;45;29;18;66;35;61;17;47;53;11;75;19;22;11;25;19;44;24;24;30;56;23;26
39;25;47;8;15;17;36;13;37;31;29;61;75;13;8;32;39;47;31;39;75;22;65;30;10;73;22;9;19;15;72;68;55;14;63;47;11;48;26;63;76;66;26;47;28
67;43;57;25;51;54;12;27;13;13;35;56;26;55;63;47;26;37;8;70;71;25;40;56;9;45;35;39;36;37;73;34;64;65;34;17;57;29;40;31;65;53;26;58;74
62;62;12;48;50;27;39;36;21;70;68;52;71;44;33;52;66;48;11;73;64;57;45;71;49;8;14;13;36;17;49;49;59;65;61;60;22;25;17;34;21;37;46;20;14
65;49;35;27;46;56;52;34;44;23;75;36;32;66;31;61;24;55;54;16;34;75;21;59;25;27;38;51;39;25;71;60;13;34;40;75;31;42;46;18;47;63;57;52;47
70;33;11;21;53;56;75;46;38;74;40;44;71;19;62;68;15;73;33;70;61;13;47;26;31;41;47;69;15;68;36;10;34;28;48;30;26;49;54;66;45;51;26;17;57
70;41;24;64;62;72;28;12;11;28;27;33;52;48;45;39;61;34;50;55;16;29;52;39;70;62;11;25;30;54;61;51;63;56;31;43;64;70;39;56;70;73;35;49;46
28;36;20;23;63;42;33;71;64;56;76;20;40;55;58;45;60;52;20;61;34;30;53;42;27;24;71;34;44;63;63;24;55;57;69;64;69;50;15;36;68;73;18;8;13
38;72;39;29;69;23;61;12;31;58;29;14;13;11;10;18;30;69;61;36;71;26;58;32;67;36;39;59;11;23;76;20;27;52;65;63;47;37;42;59;40;52;52;53;11
73;24;48;14;53;9;20;18;44;64;60;17;73;12;51;14;20;11;35;65;42;28;71;44;44;15;54;23;27;19;12;59;76;29;27;40;39;75;21;22;12;47;10;40;69
