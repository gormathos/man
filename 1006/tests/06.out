113471
76
8
